### ISSUE TITLE
Removing Command\HealtCheckCommand call to setName cause an error in symfony console.

### ISSUE THREAD

**Issue by dubaurazvan** [liip/LiipMonitorBundle#167]
```[RuntimeException]                                                               
  [Symfony\Component\Console\Exception\LogicException]                             
    The command defined in "Liip\MonitorBundle\Command\HealthCheckCommand" cann    
    ot have an empty name. 
```

**Comment by Abderrahim-WADDIZ** [comment]
Hi @dubaurazvan, the issue is related to a command 
```
Liip\MonitorBundle\Command\HealthCheckCommand 
```
to fix this issue you can set the name of the command by calling the setName function in configure because name field is required in symfony command  :

```php
$this
            ->setName('monitor:health')
            ->setDescription('Runs Health Checks')
```
I have tried create a branch and open a pull request but i don't have enough Permission to do it.

**Comment by gjuric** [comment]
Judging by the [docs](https://symfony.com/doc/3.4/console/commands_as_services.html), setting the name via a command attribute is available since Symfony 3.4/4.0 as they support lazy loading. 

Prior to that you have to specify the name.

So either the name has to be set in the class or the requirements have to be bumped to SF 3.4/4.0

**Comment by Abderrahim-WADDIZ** [comment]
@gjuric this feature must be in a major version of the bundle since it requires symfony 3.4 version
